content/en/docs/reference/glossary/cloud-controller-manager.md

```diff
@@ -19,6 +19,6 @@ with that cloud platform from components that only interact with your cluster.
 <!--more-->
 
 By decoupling the interoperability logic between Kubernetes and the underlying cloud
-infrastructure, the cloud-controller-manager component enables cloud providers to release
+infrastructure, the [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/) component enables cloud providers to release
```

Comment:
Why not update the link at line 15?

It's more noticeable in the front, isn't it?

Reply:
Till line 17 is using for glossary tooltip messages and hence created a link in the _more_ (line 19) section.

Reply:
@tamilselvan1102 The tooltip on Line 14 pertains only to the term 'control plane'. As previously suggested in a review comment, Line 15 may accommodate the inclusion of a link for 'cloud-controller-manager'.

Reply:
@dipesh-rawat 
 I have investigated,  [cloud-controller-manager] is not used anywhere as tooltip but used as [glossary_definition term_id] in the following pages. So accepted the suggestion and will update the PR.

https://kubernetes.io/docs/concepts/architecture/cloud-controller/
https://kubernetes.io/docs/concepts/overview/components/
https://kubernetes.io/docs/tasks/administer-cluster/controller-manager-leader-migration/
https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/